types/src/event.rs

```diff
@@ -127,10 +135,58 @@ impl<'de> de::Deserialize<'de> for EventKey {
             // See comment in serialize.
             #[derive(::serde::Deserialize)]
             #[serde(rename = "EventKey")]
-            struct Value<'a>(&'a [u8]);
+            struct Value(
```

Comment:
Ugly as hell. But Serde doesn't know how to process any array larger than 32 elements. Otherwise, something like `Struct Value([u8; EventKey::LENGTH])` will be much cleaner.

Reply:
That was my recollection, I just couldn't find it in their code. Glad you figured out how to work around this.

We should probably try to make a similar PR to upstream Move: https://github.com/move-language/move/blob/main/language/move-stdlib/sources/Event.move#L17

is there anyway we make this a typedef of sorts and put it at the bottom of the file -- to improve readability? Or even a macro of sort?